src/main/scala/org/wikimedia/analytics/hdfstools/HdfsRsyncCLI.scala

```diff
@@ -0,0 +1,178 @@
+// Copyright 2019 Wikimedia Foundation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package org.wikimedia.analytics.hdfstools
+
+import java.net.URI
+
+import org.apache.log4j.Level
+import scopt.OptionParser
+
+object HdfsRsyncCLI {
+
+    /**
+     * Command line options parser
+     */
+    val argsParser = new OptionParser[HdfsRsyncConfig]("HDFS-Rsync") {
+        head("HDFS-Rsync Tool", "Provides rsync equivalent to copy data between HDFS and local FS.")
```

Comment:
To keep things simple, let's assume this will be launched as 'hdfs-rsync'.  Can we use that for the doc name?  That way the help message looks good when executed by a wrapper.